integration/spark2/src/main/scala/org/apache/spark/sql/execution/strategy/StreamingTableStrategy.scala

```diff
@@ -59,7 +59,8 @@ private[sql] class StreamingTableStrategy(sparkSession: SparkSession) extends Sp
           new TableIdentifier(model.tableName, model.databaseName),
           "Alter table change datatype")
         Nil
-      case AlterTableRenameCommand(oldTableIdentifier, _, _) =>
+      case AlterTableRenameCommand(
+      oldTableIdentifier, _, _) if (isCarbonStreamingTable(oldTableIdentifier)) =>
```

Comment:
@jackylk
As per my understanding @QiangCai wanted to block the **alter table and IUD** commands for the streaming table.
The method rejectIfStreamingTable throws exception if called over the non carbon table. Therefore the alter table and IUD commands  are blocked for non carbon table.
So whether table is carbon table  should be checked first before checking the table is streaming or not.
@kushalsaha The handling is needed for the commands getting rejectected here for the 
streaming table. As per my understanding instead of adding additional if check, the rejectIfStreamingTable  method could be modified the avoid blocking the alter and IUD commands for non carbon table.

Reply:
ok, I think it is better to modify `rejectIfStreamingTable` to check only carbon table

Reply:
rejectIfStreamingTable method modified and validation check added if its  carbon Streaming Table then alter rename execution should fail and for all other cases alter rename query should execute.